src/tests/fs_tests.rs

```diff
@@ -15,6 +15,7 @@ pub mod fs_tests {
     use crate::interface::ShmidsStruct;
     pub use std::ffi::CStr as RustCStr;
     use std::mem;
+    use crate::interface::get_errno;
```

Comment:
Using `use crate::interface::{ShmidsStruct, get_errno};` would make more sense here: representing we import different structure from same file

Reply:
Updated